processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByMergingQueryRunnerV2.java

```diff
@@ -92,7 +93,8 @@ public GroupByMergingQueryRunnerV2(
       int concurrencyHint,
       BlockingPool<ByteBuffer> mergeBufferPool,
       ObjectMapper spillMapper,
-      String processingTmpDir
+      String processingTmpDir,
+      int mergeBufferSize
```

Comment:
IMO, put this next to mergeBufferPool, as it's very related.

Reply:
Moved this next to mergeBufferPool